### ISSUE TITLE
Adding dropout?

### ISSUE THREAD

**Issue by david-sg** [minimaxir/gpt-2-simple#50]
I don't see dropout as a parameter in the fine-tuning options. Would it be possible to add to help avoid overfitting?
